packages/front-end/components/OpenVisualEditorLink.tsx

```diff
@@ -98,6 +104,38 @@ const OpenVisualEditorLink: FC<{
 
       window.location.href = url;
     }
+
+    setIsNavigating(true);
+
+    let key: string;
+    try {
+      key = await getVisualEditorKey();
+    } catch (e) {
+      setIsNavigating(false);
+      setExtensionDialogText(
+        "We were unable to fetch an API key to initialize the Visual Editor. Please try again or contact support."
+      );
```

Comment:
The CTA for the extension dialog is hardcoded to "View extension".  Does that make sense for this error too?

Reply:
Not really

Reply:
Addressed